cpu/nrf52/Makefile.features

```diff
@@ -10,7 +10,7 @@ ifneq (,$(filter nrf52811xxaa nrf52820xxaa nrf52833xxaa nrf52840xxaa,$(CPU_MODEL
   FEATURES_PROVIDED += radio_nrf802154
 endif
 
-ifneq (,$(filter nrf5284%,$(CPU_MODEL)))
+ifeq (,$(filter nrf5283%,$(CPU_MODEL)))
```

Comment:
nrf52833 should be like nrf52840, no?

Reply:
Did not know, will fix then

Reply:
I can test this on microbit-v2 (nrf52833 based) later today.